ros2_control_test_nodes/resource/ros2_control_test_nodes

```diff
@@ -0,0 +1 @@
+
```

Comment:
empty file, not intended?

Reply:
It has to be there in ros2 python packages... No idea why. See, e.g., https://github.com/ros2/launch/blob/master/launch/resource/launch